changelog/unreleased/4570

```diff
@@ -0,0 +1,6 @@
+Feature: Maintain scroll position when user scrolls and navigates in file/folder list
```

Comment:
There is also a limit of characters length for the title, so I would just write the way you named the PR: `Maintain scroll position in file list`